game/src/test/scala/ScalatraBootstrapSpec.scala

```diff
@@ -0,0 +1,34 @@
+import org.scalatest._
+
+import io.aigar.model._
+import io.aigar.game._
+
+class ScalatraBootstrapSpec extends FlatSpec with Matchers {
+  def withInMemDatabase(testCode: (TeamRepository) => Any) {
+    val teamRepository = new TeamRepository(None)
+    val team1 = teamRepository.createTeam(Team(None, "secret",  "team1", 20))
+    val team2 = teamRepository.createTeam(Team(None, "secret?", "team2", 10))
+    val team3 = teamRepository.createTeam(Team(None, "secret!", "team3", 30))
+
+    try{
+      testCode(teamRepository)
+    }
+    finally {
+      teamRepository.dropSchema
+      teamRepository.closeConnection
+    }
+  }
+
+  "Creating the application" should "create a game with the players from the team repository" in withInMemDatabase { (teamRepository) =>
+    ScalatraBootstrap.fixedTeamRepository = Some(teamRepository)
```

Comment:
I might be not understanding some scala magic here, but we modify a singleton object in the tests. Shouldn't we put them `sequential`?